client/lib/mixins/analytics/index.js

```diff
@@ -63,7 +63,7 @@ const EVENTS = {
 		},
 
 		emailForwarding: {
-			addNewEmailForwardClick( domainName, mailbox, destination, success ) {
+			addNewEmailForwardClick( domainName, mailbox, destination ) {
```

Comment:
any chance we can drop the use of the mixin entirely?

Reply:
Yes! Once the PR went from huge to massive I identified a couple of tasks to follow up on, including refactoring `EmailForwardingAddNew` and `EmailForwardingDetails` to es6 classes and removing the mixing in the process.

Reply:
fair enough :)

Reply:
The plan is refactor the component that uses the mixin away from createReactClass in another PR as this one is a monster.